diff --git a/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift b/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift
index 01c36ba..02a62dc 100644
--- a/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift
+++ b/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift
@@ -375,13 +375,16 @@ public class ReactNativeDeviceActivityModule: Module {
         return
       }
       userDefaults?.set(value, forKey: key)
+      userDefaults?.synchronize()
     }
 
     Function("userDefaultsGet") { (forKey: String) -> Any? in
+      userDefaults?.synchronize()
       return userDefaults?.object(forKey: forKey)
     }
 
     Function("userDefaultsRemove") { (forKey: String) -> Any? in
+      userDefaults?.synchronize()
       return userDefaults?.removeObject(forKey: forKey)
     }
 
diff --git a/node_modules/react-native-device-activity/targets/ShieldConfiguration/Info.plist b/node_modules/react-native-device-activity/targets/ShieldConfiguration/Info.plist
index 84601bb..d96bd58 100644
--- a/node_modules/react-native-device-activity/targets/ShieldConfiguration/Info.plist
+++ b/node_modules/react-native-device-activity/targets/ShieldConfiguration/Info.plist
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIUserInterfaceStyle</key>
+	<string>Dark</string>
 	<key>REACT_NATIVE_DEVICE_ACTIVITY_APP_GROUP</key>
   <string>$(REACT_NATIVE_DEVICE_ACTIVITY_APP_GROUP)</string>
 	<key>NSExtension</key>
