diff --git a/node_modules/react-native-gesture-handler/lib/commonjs/components/ReanimatedSwipeable.js b/node_modules/react-native-gesture-handler/lib/commonjs/components/ReanimatedSwipeable.js
index f4ec7af..9612f4e 100644
--- a/node_modules/react-native-gesture-handler/lib/commonjs/components/ReanimatedSwipeable.js
+++ b/node_modules/react-native-gesture-handler/lib/commonjs/components/ReanimatedSwipeable.js
@@ -50,6 +50,7 @@ const Swipeable = /*#__PURE__*/(0, _react.forwardRef)(function Swipeable(props,
     enableTrackpadTwoFingerGesture = defaultProps.enableTrackpadTwoFingerGesture,
     dragOffsetFromLeftEdge = defaultProps.dragOffset,
     dragOffsetFromRightEdge = defaultProps.dragOffset,
+    failOffsetY,
     friction = defaultProps.friction,
     overshootFriction = defaultProps.overshootFriction,
     onSwipeableOpenStartDrag,
@@ -311,6 +312,10 @@ const Swipeable = /*#__PURE__*/(0, _react.forwardRef)(function Swipeable(props,
       dragStarted.value = false;
     });
 
+    if (failOffsetY != null) {
+      pan.failOffsetY(failOffsetY);
+    }
+
     if (!simultaneousWithExternalGesture) {
       return pan;
     }
@@ -322,7 +327,7 @@ const Swipeable = /*#__PURE__*/(0, _react.forwardRef)(function Swipeable(props,
     }
 
     return pan;
-  }, [dragOffsetFromLeftEdge, dragOffsetFromRightEdge, dragStarted, enableTrackpadTwoFingerGesture, enabled, handleRelease, onSwipeableCloseStartDrag, onSwipeableOpenStartDrag, rowState, updateAnimatedEvent, updateElementWidths, userDrag, simultaneousWithExternalGesture]);
+  }, [dragOffsetFromLeftEdge, dragOffsetFromRightEdge, failOffsetY, dragStarted, enableTrackpadTwoFingerGesture, enabled, handleRelease, onSwipeableCloseStartDrag, onSwipeableOpenStartDrag, rowState, updateAnimatedEvent, updateElementWidths, userDrag, simultaneousWithExternalGesture]);
   (0, _react.useImperativeHandle)(ref, () => swipeableMethods, [swipeableMethods]);
   const animatedStyle = (0, _reactNativeReanimated.useAnimatedStyle)(() => ({
     transform: [{
diff --git a/node_modules/react-native-gesture-handler/lib/module/components/ReanimatedSwipeable.js b/node_modules/react-native-gesture-handler/lib/module/components/ReanimatedSwipeable.js
index 3833ed6..b6242ac 100644
--- a/node_modules/react-native-gesture-handler/lib/module/components/ReanimatedSwipeable.js
+++ b/node_modules/react-native-gesture-handler/lib/module/components/ReanimatedSwipeable.js
@@ -37,6 +37,7 @@ const Swipeable = /*#__PURE__*/forwardRef(function Swipeable(props, ref) {
     enableTrackpadTwoFingerGesture = defaultProps.enableTrackpadTwoFingerGesture,
     dragOffsetFromLeftEdge = defaultProps.dragOffset,
     dragOffsetFromRightEdge = defaultProps.dragOffset,
+    failOffsetY,
     friction = defaultProps.friction,
     overshootFriction = defaultProps.overshootFriction,
     onSwipeableOpenStartDrag,
@@ -298,6 +299,10 @@ const Swipeable = /*#__PURE__*/forwardRef(function Swipeable(props, ref) {
       dragStarted.value = false;
     });
 
+    if (failOffsetY != null) {
+      pan.failOffsetY(failOffsetY);
+    }
+
     if (!simultaneousWithExternalGesture) {
       return pan;
     }
@@ -309,7 +314,7 @@ const Swipeable = /*#__PURE__*/forwardRef(function Swipeable(props, ref) {
     }
 
     return pan;
-  }, [dragOffsetFromLeftEdge, dragOffsetFromRightEdge, dragStarted, enableTrackpadTwoFingerGesture, enabled, handleRelease, onSwipeableCloseStartDrag, onSwipeableOpenStartDrag, rowState, updateAnimatedEvent, updateElementWidths, userDrag, simultaneousWithExternalGesture]);
+  }, [dragOffsetFromLeftEdge, dragOffsetFromRightEdge, failOffsetY, dragStarted, enableTrackpadTwoFingerGesture, enabled, handleRelease, onSwipeableCloseStartDrag, onSwipeableOpenStartDrag, rowState, updateAnimatedEvent, updateElementWidths, userDrag, simultaneousWithExternalGesture]);
   useImperativeHandle(ref, () => swipeableMethods, [swipeableMethods]);
   const animatedStyle = useAnimatedStyle(() => ({
     transform: [{
diff --git a/node_modules/react-native-gesture-handler/src/components/ReanimatedSwipeable.tsx b/node_modules/react-native-gesture-handler/src/components/ReanimatedSwipeable.tsx
index dc785df..1cf0de7 100644
--- a/node_modules/react-native-gesture-handler/src/components/ReanimatedSwipeable.tsx
+++ b/node_modules/react-native-gesture-handler/src/components/ReanimatedSwipeable.tsx
@@ -246,6 +246,7 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       enableTrackpadTwoFingerGesture = defaultProps.enableTrackpadTwoFingerGesture,
       dragOffsetFromLeftEdge = defaultProps.dragOffset,
       dragOffsetFromRightEdge = defaultProps.dragOffset,
+      failOffsetY,
       friction = defaultProps.friction,
       overshootFriction = defaultProps.overshootFriction,
       onSwipeableOpenStartDrag,
@@ -707,6 +708,10 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
           dragStarted.value = false;
         });
 
+      if (failOffsetY != null) {
+        pan.failOffsetY(failOffsetY as number | [number, number]);
+      }
+
       if (!simultaneousWithExternalGesture) {
         return pan;
       }
@@ -721,6 +726,7 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
     }, [
       dragOffsetFromLeftEdge,
       dragOffsetFromRightEdge,
+      failOffsetY,
       dragStarted,
       enableTrackpadTwoFingerGesture,
       enabled,
