diff --git a/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift b/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift
index 01c36ba..02a62dc 100644
--- a/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift
+++ b/node_modules/react-native-device-activity/ios/ReactNativeDeviceActivityModule.swift
@@ -375,13 +375,16 @@ public class ReactNativeDeviceActivityModule: Module {
         return
       }
       userDefaults?.set(value, forKey: key)
+      userDefaults?.synchronize()
     }
 
     Function("userDefaultsGet") { (forKey: String) -> Any? in
+      userDefaults?.synchronize()
       return userDefaults?.object(forKey: forKey)
     }
 
     Function("userDefaultsRemove") { (forKey: String) -> Any? in
+      userDefaults?.synchronize()
       return userDefaults?.removeObject(forKey: forKey)
     }
 
